test/common.js

```diff
@@ -402,8 +402,10 @@ function runCallChecks(exitCode) {
 }
 
 
-exports.mustCall = function(fn, expected) {
-  if (typeof expected !== 'number') expected = 1;
+exports.mustCall = function(fn, expected = 1) {
```

Comment:
Kind of, although it isn't actually tested unless I at least keep the fixes in test-http-response-statuscode.js. I was thinking I would clean this one up after that one lands like @targos said, so that wouldn't be an issue then.